pages/templates/index.vue

```diff
@@ -30,6 +30,62 @@ export default {
       description: template.description || dummyDescription,
     }))
 
+    categories = {
+      industry: [
+        {
+          id: 23,
+          name: 'Order Forms',
+          is_active: true,
+          created_at: '2023-10-20T13:45:21.191Z',
+          updated_at: '2023-10-20T15:42:32.427Z',
+          slug: 'order-forms',
+          kind: 'industry',
+        },
+      ],
+      generic: [
+        {
+          id: 1,
+          name: 'Donation Forms',
+          is_active: true,
+          created_at: '2023-10-20T13:45:21.030Z',
+          updated_at: '2023-10-20T15:48:55.836Z',
+          slug: 'donation-forms',
+          kind: 'generic',
+        },
+        {
+          id: 5,
+          name: 'Organization Forms',
+          is_active: true,
+          created_at: '2023-10-20T13:45:21.071Z',
```

Comment:
I believe these all were for testing purposes. Should it remain in production as well?

Reply:
No it is only for testing, once backend was deployed i remove this.